packages/protobuf-test/src/gen/js/google/protobuf/struct_pb.js

```diff
@@ -126,6 +131,9 @@ Value.prototype.toJson = function toJson(options) {
 Value.prototype.fromJson = function fromJson(json, options) {
   switch (typeof json) {
     case "number":
+      if (!isFinite(json)) {
+          throw new Error("cannot decode google.protobuf.Value from JSON " + proto3.json.debug(json));
+      }
```

Comment:
It is not possible to serialize Infinity or NaN to JSON. Under which circumstances do you expect to see a NaN or Infinity here, when parsing JSON?

Reply:
I had this there for consistency with to/fromJson, but point taken.  I removed it from the above location.